### PR TITLE
Add LADOT-operated routes

### PR DESCRIPTION
**Pull request by wmisener**
Adds two Los Angeles bus systems operated by the Los Angeles Department of Transportation, DASH and Commuter Express. They don't have individual wikidata items, but I added the operator wikidata item.

### CODE REVIEW

File: data/transit/route/bus.json
Comment (arch0345) [inline data/transit/route/bus.json:3322]:
```suggestion
      "locationSet": {"include": ["lausd_los_angeles.geojson"]},
```
This will restrict this to [just LA county](https://github.com/osmlab/name-suggestion-index/blob/main/features/us/lausd_los_angeles.geojson) in order to prevent other DASHs from being matched with this as mentioned on line 3317

File: data/transit/route/bus.json
Comment (arch0345) [inline data/transit/route/bus.json:3319]:
```suggestion
    },
```
Missing comma

File: data/transit/route/bus.json
Comment (arch0345) [inline data/transit/route/bus.json:2854]:
```suggestion
      "locationSet": {"include": ["us-ca.geojson"]},
```
This should be restricted to just California

**Review by arch0345 — CHANGES_REQUESTED** [review]
Thanks @wmisener! I've suggested some changes that should be reviewed before this gets merged

**Comment by wmisener** [comment]
I'm not sure if you're asking me to review these or a maintainer, but these all seem more than reasonable to me. If possible, I would also like to add these network:wikidata tags that I've just created: Q112028960 for DASH and Q112028995 for Commuter Express. I'm not that good at Github so I don't want to screw up the commit by trying to add them myself haha

**Comment by arch0345** [comment]
No worries, thanks for creating these!

**Review by arch0345 — APPROVED** [review]

**Pull request merged**
